sw/device/lib/crypto/drivers/entropy.c

```diff
@@ -308,14 +308,15 @@ static status_t csrng_fsm_idle_wait(void) {
   return OTCRYPTO_RECOV_ERR;
 }
 
-// Write a CSRNG command to a register.  That register can be the SW interface
+// Write a CSRNG command to a register. That register can be the SW interface
 // of CSRNG, in which case the `check_completion` argument should be `true`.
 // That register can alternatively be one of EDN's that holds commands that EDN
-// passes to CSRNG, in which case the `check_completion` argument must be
-// `false`.
+// passes to CSRNG. In this case, the check_completion register should only be
+// true for non-generate commands issued to the SW register.
```

Comment:
nit: I think here the comment refers to the argument to the function below
```suggestion
// passes to CSRNG. In this case, the check_completion argument should only be
// true for non-generate commands issued to the SW register.
```

Reply:
You're right, I'll change that.